app/controllers/static_controller.rb

```diff
@@ -0,0 +1,4 @@
+class StaticController < ApplicationController
+  def home
```

Comment:
Here is probably where we would add the skip before action: `skip_before_action :authenticate_user!`